shared/clarifai_shared/claimify/config_integration.py

```diff
@@ -167,7 +163,6 @@ def get_model_config_for_stage(config_data: Dict[str, Any], stage: str) -> str:
     if claimify_default:
         return claimify_default
 
-    # Fall back to global default
-    llm_config = config_data.get("llm", {})
-    llm_models = llm_config.get("models", {})
-    return llm_models.get("default", "gpt-3.5-turbo")
+    # Use global fallback_plugin model if no claimify-specific config
+    fallback_plugin = model_config.get("fallback_plugin", "gpt-3.5-turbo")
+    return fallback_plugin
```

Comment:
aren't we supposed to be doing away with default hard coded config values now that we have a default yaml config file that is not exposed to the user (which we are merging the user's config into)?